handlers/home.go

```diff
@@ -28,8 +29,8 @@ func Home(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if !playground.AllowWindowsInstances {
-		http.ServeFile(w, r, "./www/index-nw.html")
+	if playground.IndexFile != "" {
+		http.ServeFile(w, r, fmt.Sprintf("./www/%s", playground.IndexFile))
```

Comment:
Do we want this to have `www` as a prefix?. If we want to add volumes to the container which are not under `www` IDK if this will work. I guess we can setup the index file as `../../other_path/myindex.html` but not sure about this.